cli/lib/kontena/cli/plugins/install_command.rb

```
@@ -34,11 +35,14 @@ def execute
           rescue => ex
             $stderr.puts pastel.red("#{ex.class.name} : #{ex.message}")
             logger.error(ex)
-            spin.fail!
+            spin.fail
+            ex
           end
         end
       end
 
+      exit_with_error "Install failed: #{installed.message}" if installed.respond_to?(:message)
```

Why not just let the exception raise itself instead?